app/github_public_repo/estimate_csv_creator.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import os
+import pandas as pd
+from flask import current_app as app
+
+from app.database_etl.airtable_records_handler import get_all_records, airtable_get_request
+
+# Load env variables for Airtable request
+AIRTABLE_API_KEY = app.config['AIRTABLE_API_KEY']
+AIRTABLE_GITHUB_CSV_FIELDS_REQUEST_URL = app.config['AIRTABLE_GITHUB_CSV_FIELDS_REQUEST_URL']
```

Comment:
This is great. Do we have a process for who, on the research side, is keeping this up to date? Is this @123chrisc or have we delegated the responsibility elsewhere?

Reply:
Also, have we double checked that this contains all the fields that we want to push for analysis?

Reply:
I will talk to @123chrisc about being the DRI or assigning this responsibility!

Reply:
Resolving this conversation as Gabe will now be the maintainer of this table.